readme.md

```diff
@@ -511,6 +514,7 @@ Some of the most common codes and a possible reason include:
 - `201 Created` when a new resource is created
 - `204 No Content` when a resource is deleted
 - `400 Bad Request` when a required value is missing
+- `401 Unauthorized` when a request is made without any authorization
```

Comment:
We have `401` and `403` switched semantically, which is clearer now that you have this added. :grinning: `401` should be un*authenticated* and `403` should be un*authorized* despite HTTP saying that `401` is `Unauthorized`...one justification for having it this way is that `401` is what triggers browsers to prompt for HTTP Basic.